examples/MLIRSparseTensor/makefile

```diff
@@ -0,0 +1,5 @@
+MLIR_OPT := ../../llvm/build/bin/mlir-opt
+
+sparse-tensor-pre-rewriting-lower:
+	@${MLIR_OPT} ./sparse-tensor-pre-rewriting.mlir \
```

Comment:
Example file mismatch: `sparse-tensor-pre-rewriting.mlir` or `sparse-tensor-fuse-tensor-cast.mlir`?

Reply:
Fixed and appended into previous commit.